public/templates/index.html

```diff
@@ -42,11 +42,13 @@
       href="/assets/css/owl-carousel.css"
     />
 
-    <!-- Scripts -->
 
     <!-- jQuery -->
     <script defer src="/assets/js/jquery-2.1.0.min.js"></script>
 
+    <!-- jQuery -->
+    <scrip defer src="https://code.jquery.com/jquery-3.5.1.js"></script>
+
```

Comment:
is ths intentional?

Reply:
Yeah, 3.5.1 is the newest version of Jquery but some pages are having issues with it and others aren't. I'm unsure if this is the way to go.